src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
+
+    // Error 1: Uncommented method with misleading name
+    public int sum(int a, int b) { // Misleading method name
+        return a + b;
+    }
+
+    // Error 2: Unused private method
+    private void unusedPrivateMethod() {
```

Comment:
- The `subtract` method has an unused parameter `c`. Either remove the unused parameter or use it in the method logic, depending on the requirement.